language/sv/advancedsearchbox.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * 
+ * Advanced SearchBox [Swedish]
+ * Swedish translation by Holger (http://www.maskinisten.net)
+ * 
+ * @copyright (c) 2014 Wolfsblut ( www.pinkes-forum.de )
```

Comment:
Can you change this line to 2015? Then I don't have to modify your file at all :D
I'll merge this for the next version, thank you.